### PR TITLE
return more query subscription attributes

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

File: lib/conduit/thingspace/parsers/query_subscription.rb [hunk 1]
```diff
@@ -2,6 +2,27 @@
 
 module Conduit::Driver::Thingspace
   class QuerySubscription::Parser < Parser::Base
+
+    # Note: These are to make the result look more Sprint like
```
Comment (pedromartinez) [inline lib/conduit/thingspace/parsers/query_subscription.rb]: Do we really need this note?

File: lib/conduit/thingspace/parsers/query_subscription.rb [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+    end
+
+    attribute :activated_at do
+      last_activated_at
+    end
+
+    attribute :service_records do
+      [{ service_code: plan_code }]
+    end
+    # End sprint like attributes
```
Comment (pedromartinez) [inline lib/conduit/thingspace/parsers/query_subscription.rb]: Same

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 1 above)
Comment (dougperez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: Maybe? Someone could ask why we are returning this data. Im fine for removing cause I know why.

Comment (pedromartinez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: This doesn't really explain anything nor does it really have anything to do with Sprint.

File: lib/conduit/thingspace/parsers/query_subscription.rb [hunk 3]
```diff
@@ -37,14 +58,12 @@ class QuerySubscription::Parser < Parser::Base
...
-    # TODO: Rubyize this
     attribute :device_ids do
       device_data["deviceIds"]
     end
 
-    # TODO: Rubyize this
-    attribute :carriers do
-      device_data["carrierInformations"]
+    attribute :carrier_informations do
+      device_data["carrierInformations"] || []
```
Comment (pedromartinez) [inline lib/conduit/thingspace/parsers/query_subscription.rb]: This is within device_data?

File: lib/conduit/thingspace/parsers/query_subscription.rb [hunk 4]
```diff
@@ -2,6 +2,27 @@
...
 module Conduit::Driver::Thingspace
   class QuerySubscription::Parser < Parser::Base
+
+    # Note: These are to make the result look more Sprint like
+    attribute :plan_code do
+      return unless carrier_informations.present?
+      carrier_informations.first.to_h["servicePlan"]
+    end
+
+    attribute :status do
```
Comment (pedromartinez) [inline lib/conduit/thingspace/parsers/query_subscription.rb]: Are we doing something in code that needs this to be named `status`? Would it make sense for us to attempt to flatten `carrier_informations`?

Comment (dougperez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: Syncing looks at `status` attribute. Not sure if somethign else does.

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 3 above)
Comment (dougperez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: Yes, https://github.com/conduit/conduit-thingspace/blob/master/lib/conduit/thingspace/request_mocker/fixtures/responses/query_subscription/success.json.erb#L7

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 4 above)
Comment (pedromartinez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: I don't think we're going to be doing any syncing for this driver

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 3 above)
Comment (pedromartinez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: Ah, gotcha. That's the top level node. Thanks!

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 4 above)
Comment (dougperez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: 😿  So really all this is pointless?

File: lib/conduit/thingspace/parsers/query_subscription.rb [hunk 5]
```diff
@@ -2,6 +2,25 @@
...
+      return unless carrier_informations.present?
+      carrier_informations.first.to_h["state"]
+    end
+
+    attribute :activated_at do
+      last_activated_at
+    end
+
+    attribute :service_records do
```
Comment (pedromartinez) [inline lib/conduit/thingspace/parsers/query_subscription.rb]: What's this for?

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 4 above)
Comment (pedromartinez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: That's what I'm wondering.

File: lib/conduit/thingspace/parsers/query_subscription.rb (hunk 5 above)
Comment (dougperez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: All this is for atom needing these attributes. Same reason for this, syncing.

Comment (pedromartinez) [reply lib/conduit/thingspace/parsers/query_subscription.rb]: Ah, okay

**Pull request merged**